### DISCUSSION TITLE
Best flow to handle updating expired credit card on subscription orders

### DISCUSSION THREAD

**Discussion by y3owk1n** [swellstores/community#87] [Help] [answered]
Wondering if there's a tokenized credit card attached to an ongoing subscription, while the card was expired along the way, what is the best way to handle credit card updates?

Based on the [doc](https://developers.swell.is/frontend-api/credit-card-management#update-a-default-card), seems like updating the card will not affect the subscription. 

Where the [subscription docs](https://developers.swell.is/frontend-api/subscriptions) did not mention anything about updating the card here.

Any help would be appreciated!

**Comment by greph** [comment, marked as answer]
Hi @y3owk1n , you can update an ongoing subscription with the tokenized card (`subscription.billing.card`).

The doc makes reference to updating an account record (`account.billing.card`), which won't proactively update ongoing subscriptions associated to that account.
